tests/CMakeLists.txt

```diff
@@ -23,7 +23,7 @@ if(GMOCK_FOUND)
     cuke_add_test(integration/HookRegistrationTest)
     cuke_add_test(integration/StepRegistrationTest)
     cuke_add_test(integration/TaggedHookRegistrationTest)
-    if (NOT MSVC)
+    if(NOT WIN32)
```

Comment:
It's been there for ages and I tried several times to understand why this fails on Windows but without success. I remember it crashes but can't remember how to be honest :-) Maybe we should create an issue on GitHub and put the link in a comment to that if statement.